support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -1,12 +1,15 @@
+import { OfferBook } from 'components/offer/offer';
+
 export const productCatalog = window.guardian.productCatalog;
 
-export const productCatalogDescription = {
+export const productCatalogDescExclOffers = {
```

Comment:
Could we keep this as the canonical `productCatalogDescription`? Creating variants for tests make sense, but the idea here is that this is used through the codebase.

Reply:
Renamed this back to `productCatalogDescription`